Mirroring to GitLab is failing

### Prerequisites

- [x] I've [searched](https://github.com/accessibility-exchange/platform/issues) for existing issues

### Describe the bug

GitLab mirroring is failing, at least for Dev due to a the codebase diverging. 

```log
# git.kube.v1.colab.coop:22 SSH-2.0-GitLab-SSHD
# git.kube.v1.colab.coop:22 SSH-2.0-GitLab-SSHD
# git.kube.v1.colab.coop:22 SSH-2.0-GitLab-SSHD
# git.kube.v1.colab.coop:22 SSH-2.0-GitLab-SSHD
# git.kube.v1.colab.coop:22 SSH-2.0-GitLab-SSHD
From git.kube.v1.colab.coop:iris/iris-accessibility
 * branch              development -> FETCH_HEAD
 * [new branch]        development -> gitlab/development
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint:
hint:   git config pull.rebase false  # merge
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint:
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: Need to specify how to reconcile divergent branches.
Error: Process completed with exit code 128.
```

See: https://github.com/accessibility-exchange/platform/actions/runs/13654326754/job/38169764057

This has actually been failing since Jan 27, 2025 but was just noticed today. There was a change pushed directly to the GitLab instance on Jan 25, 2025. Which likely accounts for the diverged codebase. 

### To Reproduce

Merge any changes to dev and notice the [Mirror dev to GitLab](https://github.com/accessibility-exchange/platform/actions/workflows/mirror-dev.yml) action is failing.


### Expected behavior

The GitHub hosted branches for dev, staging, and production should be mirrored in the GitLab instances without any errors.

### Screenshots

_No response_

### Details

- Device:
- Operating system:
- Browser:
- Link to affected page:


### Additional context

_No response_

--- a/.github/workflows/mirror-dev.yml
+++ b/.github/workflows/mirror-dev.yml
@@ -32,5 +32,5 @@ jobs:
       - name: Push to GitLab
         run: |
           ssh-keyscan git.kube.v1.colab.coop >> ~/.ssh/known_hosts
-          git pull gitlab development
+          git pull gitlab development --no-rebase --no-edit
           git push gitlab dev:development

--- a/.github/workflows/mirror-production.yml
+++ b/.github/workflows/mirror-production.yml
@@ -32,5 +32,5 @@ jobs:
       - name: Push to GitLab
         run: |
           ssh-keyscan git.kube.v1.colab.coop >> ~/.ssh/known_hosts
-          git pull gitlab production
+          git pull gitlab production --no-rebase --no-edit
           git push gitlab production:production

--- a/.github/workflows/mirror-staging.yml
+++ b/.github/workflows/mirror-staging.yml
@@ -32,5 +32,5 @@ jobs:
       - name: Push to GitLab
         run: |
           ssh-keyscan git.kube.v1.colab.coop >> ~/.ssh/known_hosts
-          git pull gitlab staging
+          git pull gitlab staging --no-rebase --no-edit
           git push gitlab staging:staging

